src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -1,4 +1,13 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
+  <Target Name="OS" BeforeTargets="Build">
+    <Message Text="OS = $(OS)" Importance="High" />
+  </Target>
+  <PropertyGroup Condition="'$(OS)' != 'Unix'">
+    <TargetFrameworks>net45;netstandard1.3;uap10.0;MonoAndroid8.1;Xamarin.iOS10;netcoreapp1.0</TargetFrameworks>
```

Comment:
The TargetFrameworks property is specified in the src\MSAL.Frameworks.props - I have made changes there so I suggest you just take mine and undo these.